.github/workflows/build-pull-request.yaml

```diff
@@ -3,6 +3,35 @@ name: Validate pull request
 on:
   pull_request
 
+env:
+  GPR_USERNAME: ${{ github.actor }}
+  GPR_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
 jobs:
-  build-project:
+  validate-build:
```

Comment:
Does not make sense to name it "build-project" if it only validates (test, lint) but not builds the app (assemble)8